### PR TITLE
Create New-MpPerformanceRecording.md

### PR DESCRIPTION
**Pull request by v-smandalika**

### CODE REVIEW

File: docset/winserver2019-ps/defender/Defender.md [hunk 1]
```diff
@@ -54,4 +54,10 @@ Starts a Windows Defender offline scan.
 ### [Update-MpSignature](./Update-MpSignature.md)
 Updates the antimalware definitions on a computer.
 
+### [Get-MpPerformanceReport](./Get-MpPerformanceReport.md)
+Analyzes and reports on MDAV performance recording.
+
+### [New-MpPerformanceRecording](./New-MpPerformanceRecording.md)
+Collects a performance recording of MDAV scans
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Defender.md:61]:
```suggestion
Collects a performance recording of MDAV scans.
```

File: docset/winserver2019-ps/defender/Get-MpPerformanceReport.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+ms.date: 08/25/2021
+online version: https://docs.microsoft.com/powershell/module/defender/get-mpperformancereport?view=windowsserver2019-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Get-MpPerformanceReport
+---
+
+# Get-MpPerformanceReport
+
+## SYNOPSIS
+Analyzes and reports on MDAV performance recording
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Get-MpPerformanceReport.md:14]:
```suggestion
Analyzes and reports on MDAV performance recording.
```

File: docset/winserver2019-ps/defender/Get-MpPerformanceReport.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+```yaml
+Type: Int32
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -TopScansPerProcessPerFile
+Specifies how many top scans for output for each top process for each top file, sorted by "Duration”.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Get-MpPerformanceReport.md:120]:
```suggestion
Specifies how many top scans to output for each top process for each top file, sorted by "Duration”.
```

File: docset/winserver2019-ps/defender/Get-MpPerformanceReport.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+```yaml
+Type: Int32
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -TopScansPerProcessPerExtension
+Specifies how many top scans for output for each top process for each top extension, sorted by "Duration”.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Get-MpPerformanceReport.md:164]:
```suggestion
Specifies how many top scans to output for each top process for each top extension, sorted by "Duration”.
```

File: docset/winserver2019-ps/defender/Get-MpPerformanceReport.md [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+```yaml
+Type: Int32
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -TopScansPerFilePerExtension 
+Specifies how many top scans for output for each top file for each top extension, sorted by "Duration".
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Get-MpPerformanceReport.md:186]:
```suggestion
Specifies how many top scans to output for each top file for each top extension, sorted by "Duration".
```

File: docset/winserver2019-ps/defender/Get-MpPerformanceReport.md [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+```yaml
+Type: Int32
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -TopScansPerExtensionPerProcess 
+Specifies how many top scans for output for each top extension for each top process, sorted by "Duration”.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Get-MpPerformanceReport.md:230]:
```suggestion
Specifies how many top scans to output for each top extension for each top process, sorted by "Duration”.
```

File: docset/winserver2019-ps/defender/Get-MpPerformanceReport.md [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+```yaml
+Type: Int32
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -TopScansPerFilePerProcess 
+Specifies how many top scans for output for each top file for each top process, sorted by "Duration”.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Get-MpPerformanceReport.md:252]:
```suggestion
Specifies how many top scans to output for each top file for each top process, sorted by "Duration”.
```

File: docset/winserver2019-ps/defender/Get-MpPerformanceReport.md [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
+]
+```
+
+## DESCRIPTION
+
+The `Get-MpPerformanceReport` cmdlet analyzes a previously collected Microsoft Defender Antivirus performance recording ([New-MpPerformanceRecording](New-MpPerformanceRecording.md#new-mpperformancerecording)) and reports the file paths, file extensions, and processes that cause the highest impact to Microsoft Defender Antivirus scans.
+
+The performance analyzer provides an insight into problematic files that could cause a degradation in the performance of Microsoft Defender Antivirus. This tool is provided "AS IS" and is not intended to provide suggestions on exclusions. Exclusions can reduce the level of protection on your endpoints. Exclusions, if any, should be defined with caution.
+
+For more information on the Performance analyzer, see Performance Analyzer docs.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Get-MpPerformanceReport.md:53]:
```suggestion
For more information on the Performance analyzer, see the [Windows Performance Analyzer](windows-hardware/test/wpt/windows-performance-analyzer) topic.
```

File: docset/winserver2019-ps/defender/Get-MpPerformanceReport.md [hunk 9]
```diff
@@ -0,0 +1,265 @@
...
+
+**Supported OS versions**
+
+Windows Version 10 and higher.
+
+> [!NOTE]
+> This feature is available starting with platform version 4.18.2108.X and higher.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Get-MpPerformanceReport.md:60]:
```suggestion
Note: This feature is available starting with platform version 4.18.2108.X and higher.
```

File: docset/winserver2019-ps/defender/Get-MpPerformanceReport.md [hunk 10]
```diff
@@ -0,0 +1,265 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+## RELATED LINKS
+
+- [New-MpPerformanceRecording](New-MpPerformanceRecording.md#new-mpperformancerecording)
+- Performance Analyzer docs
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Get-MpPerformanceReport.md:265]:
```suggestion
- [Windows Performance Analyzer](windows-hardware/test/wpt/windows-performance-analyzer)
```

File: docset/winserver2019-ps/defender/New-MpPerformanceRecording.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+ms.date: 08/26/2021
+online version: https://docs.microsoft.com/powershell/module/defender/new-mpperformancerecording?view=windowsserver2019-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: New-MpPerformanceRecording
+---
+
+# New-MpPerformanceRecording
+
+## SYNOPSIS
+Collects a performance recording of MDAV scans.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/New-MpPerformanceRecording.md:14]:
```suggestion
Collects a performance recording of Microsoft Defender Antivirus scans.
```

File: docset/winserver2019-ps/defender/New-MpPerformanceRecording.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+## DESCRIPTION
+
+The `New-MpPerformanceRecording` cmdlet collects a performance recording of Microsoft Defender Antivirus scans. These performance recordings contain Microsoft-Antimalware-Engine and NT kernel process events and can be analyzed after collection using the [Get-MpPerformanceReport](Get-MpPerformanceReport.md#get-mpperformancereport) cmdlet.
+
+This `New-MpPerformanceRecording` cmdlet provides an insight into problematic files that could cause a degradation in the performance of Microsoft Defender Antivirus. This tool is provided “AS IS”, and is not intended to provide suggestions on exclusions. Exclusions can reduce the level of protection on your endpoints. Exclusions, if any, should be defined with caution.
+
+For more information on the Performance analyzer, see Performance Analyzer docs.
+
+> [!IMPORTANT]
+> This cmdlet requires elevated administrator privileges.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/New-MpPerformanceRecording.md:32]:
```suggestion
**IMPORTANT**: This cmdlet requires elevated administrator privileges.
```

File: docset/winserver2019-ps/defender/New-MpPerformanceRecording.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+
+**Supported OS versions**
+
+Windows Version 10 and higher.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/New-MpPerformanceRecording.md:36]:
```suggestion
Windows Version 10 and later.
```

File: docset/winserver2019-ps/defender/New-MpPerformanceRecording.md [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+
+> [!NOTE]
+> This feature is available starting with platform version 4.18.2108.X and higher.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/New-MpPerformanceRecording.md:39]:
```suggestion
Note: This feature is available starting with platform version 4.18.2108.X and higher.
```

File: docset/winserver2019-ps/defender/New-MpPerformanceRecording.md [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: False 
+Accept wildcard characters: False
+```
+
+## RELATED LINKS
+
+- [Get-MpPerformanceReport](Get-MpPerformanceReport.md#get-mpperformancereport)     
+- Performance Analyzer docs
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/New-MpPerformanceRecording.md:64]:
```suggestion
- [Windows Performance Analyzer](windows-hardware/test/wpt/windows-performance-analyzer)
```

File: docset/winserver2019-ps/defender/New-MpPerformanceRecording.md [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+
+
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/New-MpPerformanceRecording.md:66]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-smandalika, kindly commit the suggested changes if you agree with them. For information, PowerShell does not support markdown, so [!NOTE], etc. should be Note: <text>, etc. in PowerShell docs. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiburd** [comment]
Closing due to inactivity.

**Pull request closed without merging**